### PR TITLE
Use text instead of value in the Rapid Pro datasets

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: src/translate_rapid_pro_keys.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def remap_key_names(cls, user, data, pipeline_configuration):
                 old_key = remapping.rapid_pro_key
                 new_key = remapping.pipeline_key
                 
-                if old_key in td and new_key not in td:
+                if td.get(old_key) is not None and new_key not in td:
```
Comment (lukechurch) [inline src/translate_rapid_pro_keys.py:129]: What's the reason for this change, and the difference between the check logic of old and new keys?

**Review by lukechurch — APPROVED** [review]
LGTM, except I'd like to understand the intent of the change in src/translate_rapid_pro_keys.py

File: src/translate_rapid_pro_keys.py (hunk 1 above)
Comment (as2388) [reply src/translate_rapid_pro_keys.py:129]: Previously we were using value fields, which cannot be None.
By switching to use text fields, which can be None, the old logic allows Nones to propagate through the pipeline. Actually, we don't want those to carry through at all, because we want Nones in the raw inputs to be classed as NA.
